lib/components/Signature.js

```diff
@@ -1,12 +1,24 @@
 //@flow
 import React from 'react';
-import { GLView } from 'expo-gl';
-import { PanResponder, PixelRatio } from 'react-native';
-import { vec2 } from 'gl-matrix';
+import {
+  GLView
```

Comment:
If you end split collections with a trailing comma, when someone else comes to add to the end of the collection in future, they won't need to change the last line, and overwrite the `git blame` for it, and making the PR harder to read.